salon_creator/lib/models/salon.dart

```diff
@@ -0,0 +1,33 @@
+class Salon {
+  String salonName;
+  String media;
+  String category;
+  String description;
+  String price;
+
+  Salon({
+    this.salonName,
+    this.media,
+    this.category,
+    this.description,
+    this.price,
+  });
+
+  Map<String, dynamic> toMap() {
+    return {
+      'salonName': this.salonName,
```

Comment:
```suggestion
      'name': this.name,
```
See my comment above. Applies to the db strings as well.